corehq/apps/receiverwrapper/repeater_generators.py

```diff
@@ -41,7 +42,8 @@ def get_payload(self, repeat_record, payload_doc):
 class CaseRepeaterJsonPayloadGenerator(BasePayloadGenerator):
     def get_payload(self, repeat_record, payload_doc):
         del payload_doc['actions']
-        return json.dumps(payload_doc.get_json(lite=True))
+        data = payload_doc.get_json(lite=True)
+        return json.dumps(data, cls=DjangoJSONEncoder)
```

Comment:
i wonder if we should use this instead of [our own version](https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/web.py#L180)